deepomatic/api/exceptions.py

```diff
@@ -120,11 +107,21 @@ def get_task_id(self):
         return self.task['id']
 
 
-###############################################################################
-
-
 class HTTPRetryError(RetryError):
-    pass
+    def __str__(self):
```

Comment:
warning: renaming this could break sentry alert rules, need to pre-update those before merge?

Reply:
We could archive the previous alerts and let it create new ones ?